recipe/meta.yaml

```diff
@@ -13,14 +13,26 @@ source:
 
 build:
   number: 0
-  script: pip install --no-deps .
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - nbdime=nbdime.__main__:main_dispatch
+    - nbshow=nbdime.nbshowapp:main
+    - nbdiff=nbdime.nbdiffapp:main
+    - nbdiff-web=nbdime.webapp.nbdiffweb:main
+    - nbmerge=nbdime.nbmergeapp:main
+    - nbmerge-web=nbdime.webapp.nbmergeweb:main
+    - git-nbdifftool=nbdime.gitdifftool:main
+    - git-nbmergetool=nbdime.gitmergetool:main
+    - git-nbdiffdriver=nbdime.gitdiffdriver:main
+    - git-nbmergedriver=nbdime.gitmergedriver:main
 
 requirements:
   build:
     - python
-    - pip
+    - setuptools
   run:
     - python
+    - setuptools
```

Comment:
I'll add some tests for the `entrypoints` in another PR. Not sure how to solve the Windows issue with `pip` though.